cypress/integration/pages/articles/tests.js

```diff
@@ -31,41 +30,8 @@ const serviceHasTimestamp = service => ['news', 'urdu'].includes(service);
 
 // For testing important features that differ between services, e.g. Timestamps.
 // We recommend using inline conditional logic to limit tests to services which differ.
-export const testsThatAlwaysRun = ({ service, pageType, variant }) => {
-  describe(`Running testsToAlwaysRun for ${service} ${pageType}`, () => {
-    if (serviceHasTimestamp(service)) {
-      it('should render a formatted timestamp', () => {
-        cy.request(`${config[service].pageTypes.articles.path}.json`).then(
-          ({ body }) => {
-            const { language } = body.metadata.passport;
-            const { lastPublished, firstPublished } = body.metadata;
-            const updatedTimestamp = moment
-              .tz(lastPublished, `${appConfig[service][variant].timezone}`)
-              .locale(language)
-              .format('D MMMM YYYY');
-            const firstTimestamp = moment
-              .tz(firstPublished, `${appConfig[service][variant].timezone}`)
-              .locale(language)
-              .format('D MMMM YYYY');
-            if (lastPublished === firstPublished) {
-              cy.get('time').should('contain', updatedTimestamp);
-            } else {
-              cy.get('time')
-                .eq(0)
-                .should('contain', firstTimestamp);
-              cy.get('time')
-                .eq(1)
-                .should(
-                  'contain',
-                  `${`${appConfig[service].articleTimestampPrefix}` +
-                    ' '}${updatedTimestamp}`,
-                );
-            }
-          },
-        );
-      });
-    }
-  });
+export const testsThatAlwaysRun = ({ service, pageType }) => {
+  describe(`Running testsToAlwaysRun for ${service} ${pageType}`, () => {});
```

Comment:
As there are now no tests in this callback let's reset the description to be consistent with other page types e.g. https://github.com/bbc/simorgh/blob/3ad11b1090be4423b71f91daaa2a20e21920235a/cypress/integration/pages/frontPage/tests.js#L7-L9

Reply:
@rhenshaw56 Did you look at this?